sdk/examples/client/01_generate_addresses.rs

```diff
@@ -19,6 +19,12 @@ async fn main() -> Result<()> {
     // This example uses secrets in environment variables for simplicity which should not be done in production.
     dotenvy::dotenv().ok();
 
+    for var in ["NODE_URL", "MNEMONIC"] {
+        if std::env::var(var).is_err() {
+            panic!(".env variable '{}' is undefined, see .env.example", var);
```

Comment:
Everywhere
```suggestion
            panic!(".env variable '{var}' is undefined, see .env.example");
```

Reply:
:trollface:

Reply:
Or actually why not
`std::env::var(var).expect(&format!(".env variable '{var}' is undefined, see .env.example"));`?

Reply:
Or to calm clippy with .unwrap_or_else()